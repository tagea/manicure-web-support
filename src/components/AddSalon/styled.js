import styled from 'styled-components';
import Grid from '@material-ui/core/Grid';
import Typography from '@material-ui/core/Typography';
import GridList from '@material-ui/core/GridList';
import GridListTileBar from '@material-ui/core/GridListTileBar';
import Button from '@material-ui/core/Button';

export const FormBlockStyled = styled(Grid)`
  padding-top: 10px;
`;

export const DayStyled = styled(Typography)`
  text-transform: capitalize;
  text-align: right;
  &&{
    margin-right: 10px;
    line-height: 46px;
  }
`;

export const BlockPhotosStyled = styled(Grid)`
  overflow: hidden;
  height: 190px;
`;

export const GridListStyled = styled(GridList)`
  &&{
    flex-wrap: nowrap;
  }
  transform: translateZ(0);
`;

export const ButtonAddStyled = styled(Grid)`
  text-align: center;
  position: relative;
  top: calc(90px - 40px / 2);
`;

export const GridListTileBarStyled = styled(GridListTileBar)`
  div{
    flex-grow: 1;
  }
`;

export const InputFileStyled = styled('input')`
  display: none;
`;

export const AddPhotoStyled = styled(Button)`
  &&{
    display: inline-block;
    height: 40px;
    border-radius: 50%;
    min-width: 40px;
    width: 40px;
    padding: 9px;
  }
`;
