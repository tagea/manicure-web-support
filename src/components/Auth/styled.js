import styled from 'styled-components';
import Grid from '@material-ui/core/Grid';
import { theme } from '../../ui/Themes';

const { palette } = theme;

export const PaperStyled = styled(Grid)`
  &&{
    width: 324px;
  }
  border-radius: 4px;
  background-color: ${palette.surface.main};
  padding: 42px 27px;
`;
