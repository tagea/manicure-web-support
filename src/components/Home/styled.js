import styled from 'styled-components';
import { Typography } from '@material-ui/core';

export const TopHeaderStyled = styled(Typography)`
  flex-grow: 1;
`;
