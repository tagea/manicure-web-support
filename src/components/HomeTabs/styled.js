import styled from 'styled-components';
import Grid from '@material-ui/core/Grid';
import AppBar from '@material-ui/core/AppBar';
import Tab from '@material-ui/core/Tab';
import { theme } from '../../ui/Themes';

const { palette } = theme;

export const ContainerStyled = styled(Grid)`
  flex: auto;
  overflow: hidden;
`;

export const TabContainerStyled = styled(Grid)`
  height: 100%;
  margin-top: 30px;
`;

export const AppBarStyled = styled(AppBar)`
  &&{
    box-shadow: none;
    border-radius: 3px;
    position: relative;
    >button{
      top: 4px;
    }
  }
`;

export const TabStyled = styled(Tab)`
  &&{
    font-size: 1.2em;
    padding: 6px 0;
    color: ${palette.text.main};
  }
`;

export const LogOutButtonStyled = styled(Grid)`
  position: absolute;
  right: 20px;
  top: 4px;
`;
