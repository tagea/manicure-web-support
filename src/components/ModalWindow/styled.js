import styled from 'styled-components';
import Typography from '@material-ui/core/Typography';
import DialogTitle from '@material-ui/core/DialogTitle';
import { theme } from '../../ui/Themes';

const { palette } = theme;

export const GroupButtonStyled = styled(Typography)`
  button{
    margin: 0 4px;
  }
`;

export const DialogTitleStyled = styled(DialogTitle)`
  border-bottom: 1px solid ${palette.border.light};
  &&{
    padding-bottom: 26px;
    margin-bottom: 30px;
  }
`;
