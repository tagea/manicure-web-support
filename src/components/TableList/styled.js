import styled from 'styled-components';
import Paper from '@material-ui/core/Paper';
import TableCell from '@material-ui/core/TableCell';
import Grid from '@material-ui/core/Grid';

import { Table } from 'react-virtualized';
import { theme } from '../../ui/Themes';

const { palette } = theme;

export const PaperStyled = styled(Paper)`
  &&{
    background-color: ${palette.surface.light};
    border-radius: 3px;
    box-shadow: none;
    height: 100%;
  }
`;

export const TableStyled = styled(Table)`
  font-family: 'Roboto', sans-serif;
  font-weight: 300;
  box-sizing: border-box;
  div:focus{
      outline: none;
  }
  .rowTable{
    display: flex;
    padding: 30px 25px;
    box-sizing: border-box;
    border-bottom: 1px solid ${palette.border.light};
    border-top: 1px solid transparent;
    transition: all 0.2s ease-in;
    :hover{
      background-color: ${palette.surface.transparent};
      border-bottom: 1px solid ${palette.border.main};
      border-top: 1px solid ${palette.primary.main};
    }
  }
  .ReactVirtualized__Table__headerRow:hover{
    background-color: transparent;
    border-bottom: 1px solid ${palette.border.light};
    border-top: 1px solid transparent;
    
  }
  .headerStyle{
    padding-top: 0;
    padding-bottom: 0;
    line-height: 20px;
    color: ${palette.text.transparent};
  }
`;

export const TableCellStyled = styled(TableCell)`
  &&{
    font-size: 1.1em;
    border-bottom: none;
    padding: 0px;
  }
`;

export const TableHeader = styled(Grid)` 
  &&{
    display:flex;
    justify-content: space-between;
    margin: 0 20px;
    padding-top: 25px;
  }
`;
