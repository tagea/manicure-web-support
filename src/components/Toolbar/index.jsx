import React from 'react';
import Grid from '@material-ui/core/Grid';
import { Logo } from '../../ui/Logo';
import { AppBarStyled } from './styled';

const Toolbar = () => (
  <Grid container justify="center">
    <Grid item md={10}>
      <AppBarStyled position="static">
        <Logo />
      </AppBarStyled>
    </Grid>
  </Grid>
);

export { Toolbar };
