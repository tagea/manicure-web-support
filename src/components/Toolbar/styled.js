import styled from 'styled-components';
import AppBar from '@material-ui/core/AppBar';

export const AppBarStyled = styled(AppBar)` 
  &&{
      background-color: transparent;   
      box-shadow: none;                  
      align-items: flex-end;
      justify-content: center;
      padding-top: 10px;
  }
`;
