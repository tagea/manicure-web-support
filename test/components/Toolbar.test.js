import React from 'react';
import { shallow } from 'enzyme';
import { Toolbar } from '../../src/components/Toolbar';
import { AppBarStyled } from '../../src/components/Toolbar/styled';

it('render Toolbar component', () => {
  const ToolbarComponent = shallow(<Toolbar />);
  expect(ToolbarComponent).toMatchSnapshot();
});

it('AppBar should contain "position" property', () => {
  const ToolbarComponent = shallow(<Toolbar />);
  expect(ToolbarComponent.find(AppBarStyled).prop('position')).toBeTruthy();
});

it('AppBar should contain "position" property with "static" value', () => {
  const ToolbarComponent = shallow(<Toolbar />);
  expect(ToolbarComponent.find(AppBarStyled).prop('position')).toEqual('static');
});
