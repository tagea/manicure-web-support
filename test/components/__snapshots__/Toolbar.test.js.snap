// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`render Toolbar component 1`] = `
<WithStyles(Grid)
  container={true}
  justify="center"
>
  <WithStyles(Grid)
    item={true}
    md={10}
  >
    <styled__AppBarStyled
      position="static"
    >
      <Logo />
    </styled__AppBarStyled>
  </WithStyles(Grid)>
</WithStyles(Grid)>
`;
